contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -708,7 +708,7 @@ CREATE OR REPLACE VIEW sys.spt_tablecollations_view AS
         o.object_id         AS object_id,
         o.schema_id         AS schema_id,
         c.column_id         AS colid,
-        CASE WHEN p.attoptions[1] LIKE 'bbf_original_name=%' THEN CAST(split_part(p.attoptions[1], '=', 2) AS sys.SYSNAME)
+        CASE WHEN p.attoptions[1] LIKE 'bbf_original_name=%' THEN CAST(split_part(p.attoptions[1], '=', 2) AS sys.VARCHAR)
```

Comment:
this is new change, is there an existing test case to cover it? If not, please add one

Reply:
The execution of existing tests in pg_dump/restore github action does cover the change. So new tests are not required.

Reply:
why didn't anything changed on expected files for test? this must change certain behaviour/output and should reflect in test file change. Otherwise, we are missing relevant test

Reply:
There was a change in datatype observed dump/restore testing from varchar to text, and is not observed in JD BC/upgrade tests. Which is why after the fix there is no change needed in expected files.